crypto/multisig/src/lib.rs

```diff
@@ -134,47 +181,63 @@ impl<P: Default + PublicKey + Message> SignerSet<P> {
             return Err(SignatureError::new());
         }
 
-        // If the signature contains less than the threshold number of signers or more
-        // than the hardcoded limit, there's no point in trying.
-        if multi_sig.signatures.len() < self.threshold as usize
-            || multi_sig.signatures.len() > MAX_SIGNATURES
-        {
+        // Don't allow more than MAX_SIGNATURES signatures. This prevents potential
+        // abuse.
+        if multi_sig.signatures.len() > MAX_SIGNATURES {
             return Err(SignatureError::new());
         }
```

Comment:
I am not sure how we can identity a specific party, or what abuse is in this context. I think at some point we might want to log all SignatureErrors that get returned as a GRPC response to transaction and maybe alert on that, but I think it is low priority since there are far better ways to waste the consensus's node CPU time if someone wanted to abuse it...